repokid/cli/repokid_cli.py

```diff
@@ -307,7 +305,7 @@ def show_scheduled_roles(ctx: click.Context, account_number: str) -> None:
 @cli.command()
 @click.argument("account_number")
 @click.option("--role", "-r", required=False, type=str)
-@click.option("--all", "-a", default=False, help="Commit changes")
+@click.option("--all", "-a", default=False)
```

Comment:
shouldn't this be `is_flag=True` ?
Should we still have `help=` something?

Reply:
Nice catch!